x-pack/plugins/observability_solution/infra/public/pages/metrics/hosts/components/tabs/alerts/alerts_tab_content.tsx

```diff
@@ -141,7 +146,12 @@ const MemoAlertSummaryWidget = React.memo(
     return (
       <AlertSummaryWidget
         chartProps={chartProps}
-        featureIds={infraAlertFeatureIds}
+        ruleTypeIds={INFRA_RULE_TYPE_IDS}
+        consumers={[
+          AlertConsumers.INFRASTRUCTURE,
+          AlertConsumers.ALERTS,
+          AlertConsumers.OBSERVABILITY,
+        ]}
```

Comment:
Why the consumers here are different from the ones defined here`INFRA_ALERT_CONSUMERS`?

Reply:
Good point. I missed it. Fixed in [`7758052` (#183756)](https://github.com/elastic/kibana/pull/183756/commits/77580527b3a678aa5b9c5cc5942b8122be628aee).